scripts/deploy/localstack/ingestFiles.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# Copyright 2022-2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [ "$#" -lt 1 ]; then
+  echo "Usage: $0 <instance-id> <files>"
+  exit 1
+fi
+INSTANCE_ID=$1;
+
+THIS_DIR=$(cd "$(dirname "$0")" && pwd)
+SCRIPTS_DIR=$(cd "$THIS_DIR" && cd ../.. && pwd)
+JAVA_DIR=$(cd "$SCRIPTS_DIR" && cd ../java && pwd)
+pushd "$JAVA_DIR"
+VERSION="$(mvn -q -DforceStdout help:evaluate -Dexpression=project.version)"
+popd
+DOCKER_DIR="$JAVA_DIR/distribution/target/distribution-$VERSION-bin/scripts/docker"
```

Comment:
I think this can just reference `scripts/docker`, rather than through the distribution module.